master/internal/api/apiutils/utils.go

```diff
@@ -51,6 +52,14 @@ var (
 	}
 )
 
+// AddRBACSuffix adds a "check your permission" string to errors if RBAC is enabled.
```

Comment:
suggestion: add a comment about how this should be used. 
- endpoint can be limited by rbac
- error returned is a 404
- should not expose that the entity exists